Cargo.toml

```
@@ -95,7 +95,6 @@ tower-http = { version = "0.2.5", features = ["trace"], optional = true }
 [dev-dependencies]
 rand = "0.8.3"
 pretty_env_logger = "0.4.0"
```

```suggestion
pretty_env_logger = "0.4.0"
once_cell = "1.9.0"
```

`once_cell` is used in a doc test (alternatively you can remove it from there)

---

Actually, the failing doc test seems to be copy of the edited test, so you should probably edit the doc test, rather than adding back the dep